src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(callback, number) {
+      callback(number);
+
+      return this;
```

Comment:
check task description: Each `operate` call takes a callback and a number and sets the appropriate value to the `result` property.
you should set needed value to the result in operate method
methods for calculating the result shouldn't know anything about this or context, they just should accept needed values and provide correct result